src/fragments/lib/storage/js/get-properties.mdx

```diff
@@ -0,0 +1,49 @@
+Get the properties of the file using `getProperties` method.
+
+```typescript
+const key: string = 'srcKey';
+await Storage.getProperties(key);
+```
+
+The format of the response looks like this:
+
+```js
+ {
+    contentType: "image/jpeg",
+    contentLength: 6873,
+    eTag: "\"56b32cf4779ff6ca3ba3f2d455fa56a7\"",
+    lastModified: Wed Apr 19 2023 14:20:55 GMT-0700 (Pacific Daylight Time) {},
+    metadata: { owner : 'aws'}
+}
+
+```
+
+To utilize Encryption, the following options can be passed in to `Storage.getProperties`:
+
+```typescript
+const SSECustomerAlgorithm = 'string';
+const SSECustomerKey = new Buffer('...') || 'string';
+const SSECustomerKeyMD5 = 'string';
+
+const result = await Storage.getProperties(
+  { key: 'srcKey' },
+  {
+    SSECustomerAlgorithm,
+    SSECustomerKey,
+    SSECustomerKeyMD5
+  }
+);
+```
+
+### File Access Levels
+
+You can choose to configure [file access `level`](https://docs.amplify.aws/lib/storage/configureaccess/q/platform/js) as follows:
```

Comment:
I do think that this is valuable since it shows how this interacts with the access levels. Since the topic of access levels is covered elsewhere, could we reference that here or reuse a fragment instead of repeating the content?  Maybe this section can be as easy as a more sophisticated example with a link to where this content is detailed elsewhere?

Reply:
Thanks for the suggestion. Updated.